src/loader.js

```diff
@@ -286,13 +287,18 @@
 
             function cdnUrl(version) {
                 return url({
-                    hostname: "viz.hpccsystems.com",
+                    hostname: cdnHost,
                     port: "",
                     pathname: "/" + version + "/dist-amd"
                 });
             }
 
             return {
+                cdnHost: function(_) {
+                    if (!arguments.length) return cdnHost;
+                    cdnHost = _;
+                    return this;
```

Comment:
No "this" === the object being returned:  https://github.com/GordonSmith/Visualization/blob/a82e8d2afe2abe32a073587fd4427c49e8533110/src/loader.js#L296-L352
Its to enable the method chaining as per the unitTest